tests/assert_eq.rs

```diff
@@ -1,19 +1,18 @@
 #[allow(unused_imports)]
 use pretty_assertions::{assert_eq, assert_ne};
-extern crate difference;
 
 #[test]
 #[should_panic(expected = r#"assertion failed: `(left == right)`
 
 [1mDiff[0m [31m< left[0m / [32mright >[0m :
  Some(
      Foo {
-[31m<[0m[31m        lorem: "Hello W[0m[31mo[0m[1;48;5;52;31mr[0m[31mld!",[0m
-[32m>[0m[32m        lorem: "Hello W[0m[1;48;5;22;32mr[0m[32mo[0m[32mld!",[0m
+[31m<        lorem: "Hello W[0m[1;48;5;52;31mo[0m[31mrld!",[0m
+[32m>        lorem: "Hello Wr[0m[1;48;5;22;32mo[0m[32mld!",[0m
          ipsum: 42,
          dolor: Ok(
-[31m<[0m[31m            "hey[0m[31m",[0m
-[32m>[0m[32m            "hey[0m[1;48;5;22;32m ho![0m[32m",[0m
+[31m<            "hey",[0m
+[32m>            "hey[0m[1;48;5;22;32m ho![0m[32m",[0m
```

Comment:
Note here the change in:
- terminal codes used (to achieve the same visible styling)
- specific characters highlighted as 'different'

while providing the same information to the user.